gpt_index/playground/base.py

```diff
@@ -0,0 +1,116 @@
+"""Experiment with different indices, models, and more."""
+from __future__ import annotations
+
+import time
+from typing import Any, Dict, List, Optional, Union
+
+import pandas as pd
+from langchain.input import get_color_mapping, print_text
+
+from gpt_index.indices.base import BaseGPTIndex
+from gpt_index.indices.list.base import GPTListIndex
+from gpt_index.indices.tree.base import GPTTreeIndex
+from gpt_index.indices.vector_store.simple import GPTSimpleVectorIndex
+from gpt_index.readers.schema.base import Document
+
+DEFAULT_INDEX_CLASSES = [GPTSimpleVectorIndex, GPTTreeIndex, GPTListIndex]
+DEFAULT_MODES = ["default", "summarize", "embedding", "retrieve", "recursive"]
+
+
+class Playground:
+    """Experiment with indices, model, embeddings, modes, and more."""
+
+    def __init__(
+        self, indices: List[BaseGPTIndex], modes: Optional[List[str]] = DEFAULT_MODES
```

Comment:
why do indices not have a default here?

Reply:
These are objects that already have documents in them. Documents are specific to the user so doesn't make sense to have default here. However, I made a default option through the `from_docs` method.

Reply:
Oooooh right oops i thought this was default index classes. Actually shouldn't index_classes be specifiable by the user?

Reply:
Hm interesting. That's probably right -- I was thinking that if you were going to specify indices, you might as well just initialize normally. I'll add this in though.